src/shims/manual/Directory.Build.props

```diff
@@ -2,8 +2,13 @@
   <PropertyGroup>
     <BuildConfigurations>
       netcoreapp;
-      netcoreappaot;
       uap;
+      uapaot;
+    </BuildConfigurations>
+    <!-- System.Private.CoreLib version is different in some projects for netcoreappaot builds. -->
```

Comment:
Make sure we have an issue tracking the problems with manual shim builds.

Reply:
yes I am just writing that one

Reply:
https://github.com/dotnet/corefx/issues/36715